src/runtime/data/kata-collect-data.sh

```diff
@@ -844,6 +844,49 @@ show_latest_kata_version()
 	fi
 }
 
+show_third_party_tools()
+{
+	not_found="Tool not found"
+	start_section "Third Party Tools"
+
+	conmon_version="$(conmon --version | awk -v FS="(^conmon version)|\n" '{print $2}' | xargs || true)"
```

Comment:
I recommend to check the command exist first, otherwise the output is not so nice:
```
<summary>Third Party Tools</summary>
<p>

./kata-collect-data.sh: line 852: conmon: command not found
conmon version: Tool not found
./kata-collect-data.sh: line 856: crio: command not found
cri-o version: Tool not found
containerd version: 1.6.7
./kata-collect-data.sh: line 865: crictl: command not found
critools version: Tool not found
./kata-collect-data.sh: line 870: gperf: command not found
gperf version: Tool not found
./kata-collect-data.sh: line 874: kubectl: command not found
kubernetes version: Tool not found
libseccomp version: 2.5.1
runc version: 1.1.3

</p>
</details>

```

Reply:
Yeah. I am checking it, but I forgot to redirect stderr